common/commands.h

```diff
@@ -50,10 +50,8 @@ namespace commands::mibs {
 
 // =====================================================================================================================
 
-namespace commands::assets {
-    static constexpr const char* Subject = "assets";
-
-    class In : public pack::Node
+namespace commands {
```

Comment:
Why did you cut the namespace? This CommonIn seems used only in assets, right?

Reply:
No use in commands::discoveryauto too